cluster/addons/dns/nodelocaldns/nodelocaldns.yaml

```diff
@@ -96,8 +96,8 @@ data:
         cache 30
         reload
         loop
-        bind __PILLAR__LOCAL__DNS__ __PILLAR__DNS__SERVER__
-        forward . __PILLAR__UPSTREAM__SERVERS__ {
+        bind __LOCAL__DNS__ __DNS__SERVER__
```

Comment:
node-local-dns binary depends on this string - https://github.com/kubernetes/dns/blob/69830ba1a789b3e44d4bb37a96ef01332b8044a0/cmd/node-cache/app/configmap.go#L26

it substitutes __PILLAR__DNS__SERVER__ as the kube-dns service IP, if it is not already set. Is it ok to leave this change out or include the change in node-local-dns as well?

Reply:
ah ok gotcha, ill update that file as well thanks

Reply:
https://github.com/kubernetes/dns/pull/395 <-- assuming that should be sufficient to fix it ?

Reply:
resolving, I assume the companion PR now resolves this

Reply:
From the last discussion, we agreed to remove this diff from this PR, correct?

Reply:
ah,